crates/rattler/src/install/snapshots/rattler__install__entry_point__test__entry_point_script.snap

```diff
@@ -0,0 +1,15 @@
+---
+source: crates/rattler/src/install/entry_point.rs
+expression: script
+---
+#!/prefix\bin/python3.11
```

Comment:
there is something strange going on with the `/` vs `\` here? Or is this just an optical illusion? :)

Reply:
Since I join paths on windows I think that creates the backslash..

Reply:
but should not all path elements be backslash's in that case?

Reply:
Yep! Thats also why the tests were failing, I normalized the slashes now :)